docs/en/api/S3-API.md

```diff
@@ -9,25 +9,29 @@ priority: 0
 * Table of Contents
 {:toc}
 
-TODO: Add an introduction with examples of when to use this API
-TODO: Add list of unsupported operations and relevant JIRA tickets
-
 Alluxio supports a RESTFul API that is compatible with the basic operations of the Amazon
 [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html).
 
 The [REST API documentation](http://www.alluxio.org/restdoc/{{site.ALLUXIO_MAJOR_VERSION}}/proxy/index.html)
 is generated as part of Alluxio build and accessible through
 `${ALLUXIO_HOME}/core/server/proxy/target/miredot/index.html`.
 
-There are performance implications of using the HTTP proxy. In particular, using the proxy requires
-an extra hop. For optimal performance, it is recommended to run the proxy server and an Alluxio
-worker on each compute node. It is also recommended to put all the proxy servers behind a load balancer.
+The Alluxio S3 API should be used by applications designed to communicate with an S3-like storage,
+and would benefit from the other features provided by Alluxio, for example data caching, data
```

Comment:
The parallel grammar reads as "The Alluxio S3 API would benefit from other features provided by Alluxio", but I think you mean to refer to applications that benefit from other features.

Reply:
Updated, thanks.